README.md

```diff
@@ -1,168 +1,297 @@
+## Introduction
 
-## Example
+### What's new?
 
-```Java
-    // Instantiates MSAL Public Client App
-    PublicClientApplication myApp = new PublicClientApplication(
-                    this.getApplicationContext(),
-                    R.raw.auth_config);
+MSAL Android is now generally available with MSAL 1.0!
 
-    // Acquires a token from AzureAD 
-    myApp.acquireToken(this, SCOPES, getAuthInteractiveCallback());
+- Supported Authorities
+  - Microsoft [identity platform](https://docs.microsoft.com/en-us/azure/active-directory/develop/) (also known as the Azure Active Directory v2 Endpoint)
+  - [Azure Active Directory B2C](https://docs.microsoft.com/en-us/azure/active-directory-b2c/)
+    - [Integrate with B2C](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-andoid-b2c)
+- Microsoft [authentication broker](https://docs.microsoft.com/en-us/azure/active-directory/develop/brokered-auth) support
+  - Supports enterprise scenarios including:
+    - Device Registration
+    - Device Management
+    - Intune App Protection
+    - Device Single Sign On
+- Introduction of [Single and Multi Account](https://docs.microsoft.com/en-us/azure/active-directory/develop/single-multi-account) Public Client Applications
+- [IAccount and access to claims](https://docs.microsoft.com/en-us/azure/active-directory/develop/accounts-overview)
+- Enable Single Sign-On with different [authorization agents](https://docs.microsoft.com/en-us/azure/active-directory/develop/authorization-agents)
+- Support for synchronous methods from worker threads
+>NOTE: Tyler -  Think there may be some outdated config info on docs.ms under concepts->app scenarios->mobile app that calls web APIs. Where does new doc live?
 
-    // ...
+- Improved configuration and control of your PublicClientApplication using configuration file
+- AndroidX Compatible
 
-    // The access token can now be used to access a protected API!
-    String accessToken = authenticationResult.getAccessToken();
-```
+### Migrating from ADAL
+>NOTE: check this is right link
 
-For a full example, checkout the [code sample](https://github.com/Azure-Samples/active-directory-android-native-v2).
+You can review the ADAL to MSAL migration guide [here](https://docs.microsoft.com/en-us/azure/active-directory/develop/migrate-android-adal-msal)
```

Comment:
This is the right link, but I see that I need to update the title of the article. I'll do that and for you to match, the link text become:  To learn more about migrating your code, see the [ADAL to MSAL migration guide for Android](link)

In general, in Azure docs we don't use the 'for whatever, go [here]' pattern.  If you see it in your docs, it's a slip up.